### DISCUSSION TITLE
Upload

### DISCUSSION THREAD

**Discussion by AlbanBujar** [community/community#36707] [Other Feature Feedback, Questions, & Ideas]
Hello,

 I'm learning to use the new github. I want to upload a project from Intelij to github, it gives an error. My github account is defined in Intellij. The error says github account not found.

 Can you check my github account, is everything normal?
Or this may be the cause of the problem, if you can help, I would be very pleased.

**Comment by ossanna16** [comment]
Hey there @AlbanBujar and thank you for asking a good question! I'm glad you found our community 🙂
